packages/playground/cli/src/run-cli.ts

```diff
@@ -1,364 +1,791 @@
+			type: 'boolean',
+			default: false,
+			hidden: true,
+		})
+		.option('sitePath', {
+			describe:
+				'[DEPRECATED] Target directory with WordPress install context. This option is deprecated and will be replaced by --target-path.',
+			type: 'string',
+			hidden: true,
+		})
+
+		// @TODO: Support read-only mounts, e.g. via WORKERFS, a custom
+		// ReadOnlyNODEFS, or by copying the files into MEMFS
+		.option('mount', {
+			describe:
+				'Mount a directory to the PHP runtime. You can provide --mount multiple times. Format: /host/path:/vfs/path',
+			type: 'array',
+			string: true,
+			coerce: parseMountWithDelimiterArguments,
+		})
+		.option('mountBeforeInstall', {
+			describe:
+				'Mount a directory to the PHP runtime before installing WordPress. You can provide --mount-before-install multiple times. Format: /host/path:/vfs/path',
+			type: 'array',
+			string: true,
+			coerce: parseMountWithDelimiterArguments,
+		})
+		.option('mountDir', {
+			describe:
+				'Mount a directory to the PHP runtime. You can provide --mount-dir multiple times. Format: "/host/path" "/vfs/path"',
+			type: 'array',
+			nargs: 2,
+			array: true,
+			// coerce: parseMountDirArguments,
+		})
+		.option('mountDirBeforeInstall', {
+			describe:
+				'Mount a directory to the PHP runtime before installing WordPress. You can provide --mount-before-install multiple times. Format: "/host/path" "/vfs/path"',
+			type: 'string',
+			nargs: 2,
+			array: true,
+			coerce: parseMountDirArguments,
+		})
+		.option('blueprint', {
+			describe: 'Blueprint to execute.',
+			type: 'string',
+		})
+		.option('blueprintMayReadAdjacentFiles', {
+			describe:
+				'Consent flag: Allow "bundled" resources in a local blueprint to read files in the same directory as the blueprint file.',
+			type: 'boolean',
+			default: false,
+			hidden: true,
+		})
+		.option('quiet', {
+			describe: 'Do not output logs and progress messages.',
+			type: 'boolean',
+			default: false,
+		})
+		.option('debug', {
+			describe:
+				'Print PHP error log content if an error occurs during Playground boot.',
+			type: 'boolean',
+			default: false,
+		})
+		.option('autoMount', {
+			describe: `Automatically mount the current working directory. You can mount a WordPress directory, a plugin directory, a theme directory, a wp-content directory, or any directory containing PHP and HTML files.`,
+			type: 'boolean',
+			default: false,
+		})
+		.option('followSymlinks', {
+			describe:
+				'Allow Playground to follow symlinks by automatically mounting symlinked directories and files encountered in mounted directories. \nWarning: Following symlinks will expose files outside mounted directories to Playground and could be a security risk.',
+			type: 'boolean',
+			default: false,
+			hidden: true,
+		})
+		// Blueprint CLI options
+		.option('mode', {
```

Comment:
Could we make the name more clear?
Something like `blueprint-mode`?

Reply:
Potentially. This option name is copied frmo the `blueprints.php` runner but maybe a different name would be more useful here. Or in both places? I'm thining `--site=new|existing|none`

Reply:
It's good that `--site-type=new|existing|none` is more explicit.

Instead of `mode` to declaring an execution personality, `site-type` declares something about the site itself and lets Playground CLI judge what behavior is appropriate.